packages/easy-coding-standard/build/target-repository/.github/workflows/fixer_test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Fixer Test
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - main
+
+jobs:
+    fixer_test:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: '7.3'
+                    coverage: none
+
+            -   uses: "ramsey/composer-install@v1"
```

Comment:
It's seems checkout is still running. That cannot happen.

Also running composer install is not part of your initial test scenario. This is not testing composer require case.

Reply:
Ah you're right, with the composer require I no longer need this... Thanks!